app/Lib/lang.php

```diff
@@ -134,6 +134,7 @@
   'ct.co_people.pl' =>          'CO People',
   'ct.co_person_roles.1' =>     'CO Person Role',
   'ct.co_person_roles.pl' =>    'CO Person Roles',
+  'ct.co_petition.duplicate.check' => 'CO Petition Duplicate Check',
```

Comment:
This isn't a controller localization, this key should be elsewhere, probably `op.`.

Reply:
The lang keys have been reworked into op.match and fd.match. 
This one is "op.match.duplicate.check"